### ISSUE TITLE
Local video rotate 90 degree on Samsung i5503

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#192]
```
What steps will reproduce the problem?
1.I'm Using X-lite on PC make call to IMSDroid on Samsung I5503
2.The remote video is OK. But when i press menu->send video the local video 
rotate 90 degree.
3.I'm trying to change some code in function public void 
onOrientationChanged(int orient) of file ScreenAV.java. But local video still 
rotate 90 degree, this function only make change on remote video.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
IMSdroid  version 1.2.366 with android 2.1 update 1

Please provide any additional information below.
 I'm trying to test this on Samsung I9000. I see with front camera, the local video is OK. But when i swich to rea camera. local video continue rotate 90 degree. Please help me solve this issue. Thanks.

The detail, please attachment.


```

Original issue reported on code.google.com by `hoangdinhtrung` on 2 Mar 2011 at 8:09

Attachments:
- [imsdroid on mobile.png](https://storage.googleapis.com/google-code-attachments/imsdroid/issue-192/comment-0/imsdroid on mobile.png)
- [Xlite on PC.PNG](https://storage.googleapis.com/google-code-attachments/imsdroid/issue-192/comment-0/Xlite on PC.PNG)

**Comment by GoogleCodeExporter** [comment]
```
Can Anyone help me? Please!
```

Original comment by `hoangdinhtrung` on 3 Mar 2011 at 7:33

**Comment by GoogleCodeExporter** [comment]
```
Did you tried to enable/disable the video rotation (Options->General)?
```

Original comment by `boss...@yahoo.fr` on 3 Mar 2011 at 8:19
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Yes. I have enable video flip. But only remote video change. And i have try 
using testCamera sample from doubango. And i see the same problem.
```

Original comment by `hoangdinhtrung` on 3 Mar 2011 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
Please see attachment, when i test camera from testCamera sample.
```

Original comment by `hoangdinhtrung` on 3 Mar 2011 at 9:18

Attachments:
- [device.png](https://storage.googleapis.com/google-code-attachments/imsdroid/issue-192/comment-4/device.png)

**Comment by GoogleCodeExporter** [comment]
```
I don't think we can do somethink to fix the issue :(.
Instead you should ask on samsung forum.
Sorry.
```

Original comment by `boss...@yahoo.fr` on 3 Mar 2011 at 10:07

**Comment by GoogleCodeExporter** [comment]
```
OK, Thanks boss. But why you say that, i think may be you have some idea to 
solve this problem. That the reason make you suggest me as this from samsung 
forum. Are you mean, this problem from harware?
```

Original comment by `hoangdinhtrung` on 3 Mar 2011 at 10:15

**Comment by GoogleCodeExporter** [comment]
```
i think the issue is similar to the original problem that I found with HTC. In 
this case, rear camera was ok, but front was rotated 90. I believe this is due 
to how the hardware manufacturer installs their camera sensors. A few things 
you can try to do, but they would be specific to your type of the device. Flip 
camera flag in the code is done for the front camera. if you don't enable the 
flag and front is working properly, then you will need to change the code and 
enable rotation on back facing camera only. If rear facing camera only works 
with Flip flag  set, then you will need to enable the same code for both 
cameras on your device. 
```

Original comment by `alex9...@gmail.com` on 10 Apr 2011 at 2:08

**Comment by GoogleCodeExporter** [comment]
```
Dear alex9...@gmail.com,
Thanks so much. I also think like you. Maybe we do not have to edit multiple 
video signals received, so that by default it will display properly. It does 
not concern the video signal is transmitted like? Just the right signal that 
the device automatically will receive images for accurate results.
```

Original comment by `hoangdinhtrung` on 13 Apr 2011 at 3:48

**Comment by GoogleCodeExporter** [comment]
```
Fixed by "imsdroid-2.0.419-preview".
```

Original comment by `boss...@yahoo.fr` on 5 Jun 2011 at 5:27
- Changed state: **Fixed**
